cli/features/tutorial.feature

```diff
@@ -159,14 +160,17 @@ Feature: Following the tutorial
 
       dependencies:
         mongo:
-          version: '3.4.0'
-          docker_flags:
-            volume: '-v {{EXO_DATA_PATH}}:/data/db'
-            online_text: 'waiting for connections'
-            port: '-p 27017:27017'
+          dev:
```

Comment:
As we discussed, we will list dependencies and their meta data in `application.yml`. Services will just list `mongo3.4.0` or something as a dependency. Then a user could even run `exo add #{dependency-name}` to add a dependency to `application.yml`. @kevgo can you write the ticket for this? I want to make sure all the details and requirements are correct.